.github/workflows/devtools.yml

```diff
@@ -78,6 +78,37 @@ jobs:
           ${{ github.workspace }}/.gclient
```

Comment:
doesn't test/e2e also need to be here?

Reply:
No, there's no build artifacts in this folder.

But it should be added to `generate-devtools-hash.sh`